pkg/sentry/certs/store.go

```diff
@@ -12,10 +12,7 @@ import (
 	"github.com/dapr/dapr/pkg/sentry/config"
 	"github.com/dapr/dapr/pkg/sentry/consts"
 	"github.com/dapr/dapr/pkg/sentry/kubernetes"
-)
-
-const (
```

Comment:
@DeepanshuA Please revert the changes to this file alone. 
Have the utils function ... but use the local var and local function defined in this file and not the utils one.

Reply:
Accepting a param as defaultNamespace now in utils.